src/Protos/Grpc/cluster.proto

```diff
@@ -0,0 +1,138 @@
+﻿syntax = "proto3";
+package event_store.cluster;
+
+import "shared.proto";
+
+service Cluster {
```

Comment:
It might be better to split this service into Gossip {read, update} and Election{viewchange, ...}  that will eliminate Cluster.Cluster which is a pet peeve.